.github/templates/linux_ci_workflow.yml.j2

```diff
@@ -141,6 +141,22 @@ jobs:
             -w /var/lib/jenkins/workspace \
             "${DOCKER_IMAGE}" \
             sh -c 'sudo chown -R jenkins . && .jenkins/pytorch/build.sh'
+      - name: Display and upload binary build size statistics (Click Me)
+        # temporary hack: set CIRCLE_* vars, until we update
+        # tools/print_test_stats.py to natively support GitHub Actions
+        env:
+          SCRIBE_GRAPHQL_ACCESS_TOKEN: ${{ secrets.SCRIBE_GRAPHQL_ACCESS_TOKEN }}
+          CIRCLE_BRANCH: ${{ steps.parse-ref.outputs.branch }}
```

Comment:
nit: Eventually we should just get rid of the vendor specific env variables here and make it so that the script itself just has it's own specific env variables it reads from that we can populate

Reply:
yup. will create another PR to address all these